webclient/src/common.i18n.json

```diff
@@ -1,6 +1,27 @@
 {
   "Common": {
     "language": "Translate into English.",
-    "disconnect": "Disconnect"
+    "disconnect": "Disconnect",
+    "label": {
+      "confirmPassword": "Confirm Password",
+      "confirmSure": "Are you sure?",
+      "country": "Country",
+      "delete": "Delete",
+      "email": "Email",
+      "hostName": "Host Name",
+      "hostAddress": "Host Address",
+      "password": "Password",
+      "port": "Port",
+      "realName": "Real Name",
+      "saveChanges": "Save Changes",
+      "token": "Token",
+      "username": "Username"
+    },
+    "validation": {
+      "minChars_one": "Minimum of {{count}} character required",
+      "minChars_other": "Minimum of {{count}} characters required",
```

Comment:
@ebbit1q here's your plural example.

Reply:
this does not look like the example in transifex's docs https://docs.transifex.com/formats/json

Reply:
tl/dr it seems like we'll be fine, i dont see a way to reconcile i18n-next format with either ICU or Structured json formats in Transfex.

From their docs:
> Example:
>
> "files" : "{count, plural, one {You have {count} file.} other {You have {count} files.}}"
>
> For pluralized strings that follow a different structure like:
>
> "string" : "{number, plural, =1 {1 New}, =2 {# New}}"
>
> the Structured JSON parser will handle them as simple entries in the editor as follows:

![Structured_JSON_Plurals_Support](https://user-images.githubusercontent.com/7624403/155906808-032507dd-0ee4-4802-b37f-7582a4eda8a8.jpeg)

Reply:
What does this mean for us operationally?  Will translations need to be done twice per language?

Reply:
We'll need to provide both either way. `i18n-next` wants the following:
```
"key_one": "Minimum of {{count}} character required",
"key_other": "Minimum of {{count}} characters required",
```

Transifex wants this:
```
"key": "{prop, plural, one {{prop} translation} other {{prop} translations}}"
```

I belive Transifex with help the translators by showing them as group vs separate translations, which would be a bummer to lose, but not the end of the world.

Reply:
Oh, it looks like `i18n-next` has a plugin for us to use the ICU format. I'll look into adding this in with this PR.

https://react.i18next.com/misc/using-with-icu-format

Reply:
we will see how it performs when this is merged, translations are picked up automatically